src/pages/[lang]/it-software-solutions-for-business.astro

```diff
@@ -1,32 +1,38 @@
----
-import { t } from "i18next";
-import SkillPage from "../../layouts/SkillPage.astro";
-import { getStaticLangPaths, updateLang } from "../../routing/lang";
-
-export const getStaticPaths = getStaticLangPaths;
-updateLang(Astro.url.pathname);
----
-
-<SkillPage
-  skillNumber="09"
-  skillName="IT Software Solutions for Business"
-  img="https:////live.staticflickr.com/65535/52434272561_5e10bb3507_h.jpg"
-  nationalTeam={[]}
-  timelineEntries={["wsc2024", "einsteinCup2024", "euroskills2023", "nationalChampionship2023", "wsc2022", "einsteinCup2022"].map((entryId) => ({
-    title: t(`skills.09.timeline.${entryId}.title`),
-    date: t(`skills.09.timeline.${entryId}.date`),
-    text: t(`skills.09.timeline.${entryId}.text`),
-  }))}
-  additionalLinks={[
-    {
-      name: "worldskills.org",
-      text: t("skills.09.links.wsi"),
-      href: "https://worldskills.org/skills/id/221/",
-    },
-    {
-      name: "worldskills2024.com",
-      text: t("skills.09.links.wsc2024"),
-      href: "https://worldskills2024.com/en/skills/it-software-solutions-for-business",
-    },
-  ]}
-/>
+---
+import { t } from "i18next";
+import SkillPage from "../../layouts/SkillPage.astro";
+import { getStaticLangPaths, updateLang } from "../../routing/lang";
+import { load_history } from "../../utils/async_helper";
+
+export const getStaticPaths = getStaticLangPaths;
+updateLang(Astro.url.pathname);
+
+const { lang } = Astro.params;
+
+const history_data = await load_history("skill09", lang);
```

Comment:
I hope we can remove this line (see review comment on json loading) but just FYI, in JS it is very common to use camelCase instead of snake_case.